### ISSUE TITLE
No bounding boxes

### ISSUE THREAD

**Issue by u221** [Bellspringsteen/OurCamera#12]
I trained for 1400 steps and yet I am unable to see the bonding boxes on eval in tensor board, can you help me with this ?

**Comment by u221** [comment]
@aaronbauman can you help ?

**Comment by aaronbauman** [comment]
I'm afraid I didn't get it working either.
The training data that's bundled here is specific to the particular corner in which the author was concerned.
Training for a new camera would require manually tagging a new set of traning data, and I haven't found the right tools to do it.

**Comment by u221** [comment]
@Bellspringsteen I trained for 1400 steps and yet I am unable to see the bonding boxes on eval in tensor board, can you help me with this

**Comment by Bellspringsteen** [comment]
Hello,
So you are running against the same data that i provided? With the same base model? 
If you stop the run and create a python object_detection/export_inference_graph.py \
    --input_type image_tensor \
    --pipeline_config_path /path/data/models/faster_rcnn_resnet101_cars.config \
    --trained_checkpoint_prefix /path/data/models/model/train/model.ckpt-18557 \
    --output_directory output_inference_graph.pb frozen output graph and then run the eval script on one of the images do you see bounding boxes? You can also lower the threashold for what creates a box to say .50. You should definitely see boxes then (thought they may not be accurate.) Let me know if you want me to be more descriptive.

Also, for training on your own images, this model works ok for overhead lower res cameras like the feed from http://dotsignals.org/ but you are right that you might have to retrain with other data if you want to use other types/positions of cameras.

**Comment by u221** [comment]
@Bellspringsteen Yes I am training with the same data and same base model.  I could not get any bounding boxes, can you please be more descriptive and also should I train it for like 18557 steps to get bounding boxes ?

**Comment by u221** [comment]
@Bellspringsteen Yes I am training with the same data and same base model.  I could not get any bounding boxes, can you please be more descriptive and also should I train it for like 18557 steps to get bounding boxes ?
